### PR TITLE
#82 Get weather calendar data

### PR DESCRIPTION
**Pull request by Edhoru**
These two new commits add the functionality to retrieve the information for the weather calendar in the app, both Settings and Annotations.

Both new queries include it's own Test

### CODE REVIEW

File: ApiClient/Source/Weather/Models/Protocols/WeatherAnnotation.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+import Foundation
+
+public protocol WeatherAnnotation {
+    
+    var fo: Int { get }
```
Comment (dmfs) [inline ApiClient/Source/Weather/Models/Protocols/WeatherAnnotation.swift]: Can we give those more descriptive names?

File: ApiClient/Source/Weather/Models/Protocols/WeatherSettingsItem.swift [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+//
+
+import Foundation
+
+public struct WeatherSettingsItem: Decodable {
+    ///The unit set as default when subscribing to the Weather calendar
+    var _default: String
+    ///The category display name for the Setting
+    var title: String
+    ///An array of all the posible units a user can choose for the Subscription to the calendar
```
Comment (dmfs) [inline ApiClient/Source/Weather/Models/Protocols/WeatherSettingsItem.swift]: Please add an explanation for why this is a dictionary.

**Comment by dmfs** [comment]
Looks good to me. I have just two minor nitpicks.

File: ApiClient/Source/Weather/Models/Protocols/WeatherAnnotation.swift (hunk 1 above)
Comment (Edhoru) [reply ApiClient/Source/Weather/Models/Protocols/WeatherAnnotation.swift]: done

File: ApiClient/Source/Weather/Models/Protocols/WeatherSettingsItem.swift (hunk 2 above)
Comment (Edhoru) [reply ApiClient/Source/Weather/Models/Protocols/WeatherSettingsItem.swift]: It's a dictionary, not an array, the comment was wrong.

Done

**Pull request merged**